Documents/panel.md

```diff
@@ -21,6 +33,14 @@ sbt> clean
 sbt> assembly
 ```
 Then the panel loader can be run with `java -jar  panel/target/scala-2.12/panel.jar path/to/institution_file.csv`
+
+## Error codes
+There are four ways the panel loader can fail.  The exit code and error message should tell you what happened.
+1) There were no command line arguments passed to the loader
+2) The path passed to the loader didn't point to a file
+3) The call to `institutions/create` didn't return the correct response.  This can indicate that you don't have the correct environment variables set, or that something is wrong with the hmda-platform.
+4) The loader didn't finish processing all the institutions.  This will happen when running the real panel file, but unsure as to why this happens.
```

Comment:
I'm still trying to figure out where the bug is (so I can create the ticket in the appropriate repo)